### DISCUSSION TITLE
Could I ask what this parameter(buffer_response) does?

### DISCUSSION THREAD

**Discussion by togo-hst** [apache/couchdb#5148] [Q&A] [answered]
Hello Nickva @nickva , Hello Robert @rnewson ,

I am very sorry to bother you two. 
We have been using CouchDB products. However, we upgraded the version(`v3.3.3`) of CouchDB last month and found that the application had abnormal access to data. Thank you very much for your help and quickly find out  the problem of this parameter`buffer_response` https://github.com/apache/couchdb/issues/5094 . 
Although this problem has been solved, I would like to know what impact it will have if this parameter(`buffer_response`) is set to `true` or `false`?  [I can only find the explanation here currently](https://docs.couchdb.org/en/stable/config/http.html#chttpd/buffer_response).

We have 7 databases, only 1 of them(`sw360changelogs`) cannot replicate data at the time.

Can I know more detailed explanation behind this parameter? Thanks a lot!

**Comment by rnewson** [comment, marked as answer]
The linked documentation explains it all, I think? if buffered_response is true couchdb will calculate the whole response before starting the http response. this interacts very badly with _changes?feed=continuous as that has no "end".

I think we will deprecate and remove the option, and we do not recommend you enable it.

**Reply by togo-hst** [reply]
I got it , Thank you so much! Robert!

Yes, unfortunately this configuration was enabled by me while researching CouchDB, but the testing did not cover it. Eventually, it was pushed to the production environment and the customer discovered the issue.

**Reply by togo-hst** [reply]
Does couchdb-lucene-v2.1.0 provide some configuration parameters that can also be solved, either?

**Reply by togo-hst** [reply]
Or I misunderstood, according to what you meant, it should be like a streaming operation that can never be calculated and completed

**Reply by rnewson** [reply]
couchdb-lucene is obsolete at this point, it does not work with couchdb 2.0 or higher since clustering was added. Use the built-in search feature  https://docs.couchdb.org/en/stable/ddocs/search.html instead.

**Reply by togo-hst** [reply]
Thank you Robert! I will try this plugin for search of lucene. hava a good day! 🤝
